libs/blog/shared/ui-social-media-icons/src/lib/social-media-icons.component.ts

```diff
@@ -1,10 +1,10 @@
 import { ChangeDetectionStrategy, Component } from '@angular/core';
 
-import { IconComponent, IconType } from '@angular-love/blog/shared/ui-icon';
+import { IconComponent } from '@angular-love/blog/shared/ui-icon';
 
 export type IconConfig = {
   href: string;
-  name: IconType;
+  name: string;
```

Comment:
Yes, the only way to create a `IconType` is to hardcode it, but I'm not totally sure about it. When new icon is added, the type has to be updated. It would be great if we could avoid forcing the developer to extend the type.

Reply:
Appended but I'm not absolutely sure about this